lib/system/sysstr.nim

```diff
@@ -304,20 +304,10 @@ proc setLengthSeq(seq: PGenericSeq, elemSize, elemAlign, newLen: int): PGenericS
     when not defined(boehmGC) and not defined(nogc) and
          not defined(gcMarkAndSweep) and not defined(gogc) and
          not defined(gcRegions):
-      when false: # deadcode: was used by `compileOption("gc", "v2")`
```

Comment:
Looks like an unrelated improvement that I don't understand. Please explain.

Reply:
The GitHub diff cannot show the actual changes reliably.